docs/admin/templates/extending-templates/prebuilt-workspaces.md

```diff
@@ -25,120 +34,120 @@ template parameters, Coder assigns an existing instance, reducing setup time sig
        memory        = 16
      }
      prebuilds {
-       instances = 3
+       instances = 3  // Number of prebuilt workspaces to maintain
      }
    }
    ```
 
-1. Publish and import the template.
-1. Coder automatically provisions another prebuilt workspace through an internal reconciliation loop
-   (similar to Kubernetes) to maintain the number of specified `instances`.
+After you create or update the template, Coder automatically provisions and maintains prebuilt workspaces through an
+internal reconciliation loop (similar to Kubernetes) to maintain the number of specified `instances`.
 
-## Workspace ownership
+Prebuilt workspaces are displayed in the list of workspaces on the Coder dashboard with the owner set to `prebuilds`.
 
-After a prebuilt workspace is created, it is owned by the unprivileged pseudo-user `prebuilds`, which belongs to the
-`Everyone` group.
-Coder uses the `prebuilds` user to identify unclaimed prebuilt workspaces.
-You can add the `prebuilds` user to additional groups if you need to.
+## Prebuild lifecycle
 
-## View prebuilt workspaces
+Prebuilt workspaces follow a specific lifecycle from creation through eligibility to claiming.
 
-You can view prebuilt workspaces in the **Workspaces** view in the Coder dashboard:
+Expand each item in this list for more information about the stage:
 
-![A prebuilt workspace in the dashboard](../../../images/admin/templates/extend-templates/prebuilt/prebuilt-workspaces.png)
+1. <details><summary>Prebuilds provisioning and initial state</summary>
+
+   After you configure a preset with Prebuilds and publish the template:
+
+   1. Coder automatically creates prebuilt workspaces up to the specified `instances` count.
+   1. Each new prebuild is initially owned by an unprivileged system pseudo-user named `prebuilds`.
+      - The `prebuilds` user belongs to the `Everyone` group (you can add it to additional groups if needed).
+   1. Each Prebuild receives a randomly generated name for identification.
 
-## Claiming
+   </details>
 
-A prebuilt workspace is automatically and transparently assigned to a user when a:
+1. <details><summary>Eligibility process</summary>
 
-- User creates a new workspace via the API or the Coder web UI.
-- User selected a preset which has been configured for prebuilds.
-- Prebuilt workspace is in an eligible state.
+   Before a prebuilt workspace is available to users:
 
-The ownership of the prebuilt workspace will change to the requesting user, and this is referred to as a "claim".
+   1. The workspace is provisioned like a regular workspace.
+   1. The workspace reaches `running` state.
+   1. The agent connects and reports `ready` status.
+   1. All bootstrap procedures and startup scripts complete successfully.
+   1. The workspace is marked as `eligible` to be claimed.
```

Comment:
```suggestion
```

There is no such marking.

Reply:
I guessed this from the [available metrics](https://github.com/coder/coder/pull/17580/files#diff-c18bc8ccd336eb66c2307125b15bd13b19df2d190cae3939075ac741ab5cc19cR207)